packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts

```diff
@@ -0,0 +1,485 @@
+        const inputAMountMax = Number(state.components.simpleBuy.pair.buyMax) / 100
+        const outputCurrency = state.components.simpleBuy.cryptoCurrency
+
+        analytics.push(AnalyticsKey.BUY_AMOUNT_ENTERED, {
+          id,
+          input_amount: inputAmount,
+          input_currency: inputCurrency,
+          max_card_limit: inputAMountMax,
+          nabuId,
+          originalTimestamp: getOriginalTimestamp(),
+          output_currency: outputCurrency,
+          platform: 'WALLET',
+          type: AnalyticsType.EVENT
+        })
+        break
+      }
+      case AT.components.simpleBuy.HANDLE_SB_MAX_AMOUNT_CLICK: {
+        const state = store.getState()
+        const maxCardLimit = Number(action.payload.amount) / 100
+        const inputCurrency = state.components.simpleBuy.fiatCurrency
+        const outputCurrency = state.components.simpleBuy.cryptoCurrency
+
+        analytics.push(AnalyticsKey.BUY_AMOUNT_MAX_CLICKED, {
+          id,
+          input_currency: inputCurrency,
+          max_card_limit: maxCardLimit,
+          nabuId,
+          originalTimestamp: getOriginalTimestamp(),
+          output_currency: outputCurrency,
+          platform: 'WALLET',
+          type: AnalyticsType.EVENT
+        })
+        break
+      }
+      case AT.components.simpleBuy.HANDLE_SB_MIN_AMOUNT_CLICK: {
+        const state = store.getState()
+        const inputCurrency = state.components.simpleBuy.fiatCurrency
+        const outputCurrency = state.components.simpleBuy.cryptoCurrency
+
+        analytics.push(AnalyticsKey.BUY_AMOUNT_MIN_CLICKED, {
+          id,
+          input_currency: inputCurrency,
+          nabuId,
+          originalTimestamp: getOriginalTimestamp(),
+          output_currency: outputCurrency,
+          platform: 'WALLET',
+          type: AnalyticsType.EVENT
+        })
+        break
+      }
+      case AT.components.simpleBuy.HANDLE_SB_METHOD_CHANGE: {
+        const paymentType = action.method.type
+
+        analytics.push(AnalyticsKey.BUY_PAYMENT_METHOD_SELECTED, {
+          id,
+          nabuId,
+          originalTimestamp: getOriginalTimestamp(),
+          payment_type: simpleBuyPaymentTypeDictionary(paymentType),
+          platform: 'WALLET',
+          type: AnalyticsType.EVENT
+        })
+        break
+      }
+      case AT.components.simpleBuy.ADD_CARD_DETAILS_FAILURE: {
+        const state = store.getState()
+        const cardType =
+          getCardTypeByValue(state.form.addCCForm.values['card-number'])?.type || 'NOT_KNOWN'
+        const countryBilling = state.form.addCCForm.values.billingAddress.country
+        const reason = state.form.addCCForm.error
+
+        analytics.push(AnalyticsKey.CARD_REJECTED, {
+          card_type: cardType,
+          country_billing: countryBilling,
+          id,
+          nabuId,
+          originalTimestamp: getOriginalTimestamp(),
+          product: 'SIMPLE_BUY',
```

Comment:
line 173 you called it `SIMPLEBUY`. this probably needs to be consistent

Reply:
It was removed from the other one